gh-repo-visibility

```diff
@@ -576,18 +669,21 @@ if [[ -n "${INPUT_FILE_NAME}" ]]; then
     #####################################
     UpdateRepoVisibility "${DESTINATION_ORG_NAME}" "${DESTINATION_REPO_NAME}" "${VISIBILITY}" "${TEMPLATE}"
   done < "${INPUT_FILE_NAME}"
-else
-  #########################################
-  # Get the list of all repos in the orgs #
-  #########################################
-  GetAllSourceRepos
-  GetAllDestinationRepos
-  ###############################################################
-  # Loop through the repos and generate list of repos to update #
-  ###############################################################
-  GenerateUpdateList
+}
+################################################################################
+#### Function ParseUpdateArray #################################################
+ParseUpdateArray() {
+  # Check api limit
+  CheckAPILimit
+  # Ser var for API limit used
+  API_LIMIT_REMAINING="${API_REMAINING}"
   # Loop through the list of repos and update the visibility
   for OBJECT in "${UPDATE_REPO_ARRAY[@]}"; do
+    # Check api limit remainging
+    if [ "${API_LIMIT_REMAINING}" -lt 1 ]; then
```

Comment:
could be `-eq 0` Just saying 😄

Reply:
yeah good point
lol that was github copiliot... its craxzy